styles/components/ItemTable.module.scss

```diff
@@ -18,23 +18,28 @@
       line-height: var(--nypl-lineHeights-body-body2);
     }
 
+    th {
+      font-size: var(--nypl-fontSizes--2);
```

Comment:
I thought the latest `Table` update included this?

Reply:
The default font size is larger but Apoorva requested the override. Maybe this should be the default in the DS?

Reply:
`--nypl-fontSizes--2` is 12px. It looks like you have the `tableTextSize` prop set to `"body2"`, so the font size for the column headings should already be 12px be default.

Reply:
I believe i was missing the tableTextSize prop, I just added it and was able to get rid of the font-size in the CSS file.